### PR TITLE
ci: build windows .zip

### PR DESCRIPTION
**Pull request by JohnMcLear**
Tests building the .zip on linux then passes that to windows and windows runs it and makes sure it can open a pad.

### CODE REVIEW

File: .github/workflows/windows-zip.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    - name: Run the backend tests
+      run: cd src && npm test
+
+    - name: Build the .zip
+      run: src/bin/buildForWindows.sh
+
+    - name: Archive production artifacts
+      uses: actions/upload-artifact@v2
+      with:
+        name: e
```
Comment (rhansen) [inline .github/workflows/windows-zip.yml:42]:
```suggestion
        name: etherpad-lite-win.zip
```

File: .github/workflows/windows-zip.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    name: Deploy Windows Zip on Windows
+    needs: build
+    runs-on: windows-latest
+
+    steps:
+    - name: Download all workflow run artifacts
+      uses: actions/download-artifact@v2
+
+    - name: Extract Etherpad
+      run: 7z x e\etherpad-lite-win.zip -oetherpad
```
Comment (rhansen) [inline .github/workflows/windows-zip.yml:61]:
```suggestion
    steps:
    - name: Download zip
      uses: actions/download-artifact@v2
      with:
        name: etherpad-lite-win.zip

    - name: Extract Etherpad
      run: 7z x etherpad-lite-win.zip -oetherpad
```

File: .github/workflows/windows-zip.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+    - name: list
+      run: dir etherpad
+
+    - name: Run Etherpad
+      run: |
+            cd etherpad
+            node node_modules\ep_etherpad-lite\node\server.js &
```
Comment (rhansen) [inline .github/workflows/windows-zip.yml:69]: Ampersand is a POSIX shell script thing, and IIUC the [default shell for Windows is PowerShell Core](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell). Does ampersand also work with PowerShell?

File: .github/workflows/windows-zip.yml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+    - uses: actions/setup-node@v2
+      with:
+        node-version: 12
+
+    - name: Install libreoffice
+      run: |
+        sudo add-apt-repository -y ppa:libreoffice/ppa
+        sudo apt update
+        sudo apt install -y --no-install-recommends libreoffice libreoffice-pdfimport
```
[the remark this answers is not in the document]
Comment (JohnMcLear) [reply .github/workflows/windows-zip.yml:28]: No

File: .github/workflows/windows-zip.yml (hunk 3 above)
Comment (rhansen) [reply .github/workflows/windows-zip.yml:69]: [Yup it does.](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_operators?view=powershell-7.1#background-operator-)

Comment (JohnMcLear) [reply .github/workflows/windows-zip.yml:69]: Is it even powershell?  If so.  Yes.  I just stack overflowed my way through this feeling dirty at every step ..

Comment (rhansen) [reply .github/workflows/windows-zip.yml:71]: Yes, it is PowerShell Core.

File: .github/workflows/windows-zip.yml (hunk 4 above)
[the remark this answers is not in the document]
Comment (rhansen) [reply .github/workflows/windows-zip.yml:28]: OK. Please remove.

File: .github/workflows/windows-zip.yml [hunk 5]
```diff
@@ -0,0 +1,66 @@
+name: "Build Windows Zip"
```
Comment (rhansen) [inline .github/workflows/windows-zip.yml:1]:
```suggestion
name: "Windows Zip"
```

**Review by rhansen — APPROVED** [review]

**Pull request merged**